.github/DISCUSSION_TEMPLATE/resources.yml

```diff
@@ -0,0 +1,49 @@
+title: "<resource title>"
+labels: ["Resource"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ## Resource Submission
+        
+        Please provide the following details about the resource you are sharing. 
+        
+        Note that duplicate resources may be removed to keep things tidy.
+  - type: textarea
+    id: context
+    attributes:
+      label: Context
+      description: 'Provide any additional context about the resource. Why is it useful? Who might it benefit?'
+      placeholder: 'This guide is useful for those looking to understand React from scratch.'
+    validations:
+      required: true
+  - type: input
+    id: url
+    attributes:
+      label: URL
+      description: Provide the most direct link to the resource
+      placeholder: 'https://example.com'
+    validations:
+      required: true
+  - type: textarea
+    id: extra
+    attributes:
+      label: Additional information
+      description: "Share any additional links or information here."
+      value: |
+        ...
+    validations:
+      required: false
```

Comment:
Do we need this? Extra fields will probably encourage filling them, and may lead to unnecessary repeat of information that should be in the main text box.

Reply:
Removed in https://github.com/epiverse-trace/epiverse-trace.github.io/pull/234/commits/b6aada136b99876235959f2808dc958fe3520407